Update ManageLedgerParameters docs (#2886)
dfinity/portal: docs/developer-docs/daos/sns/managing/making-proposals.mdx

### `ManageLedgerParameters`
The proposal `ManageLedgerParameters` can be used to update some of the SNS ledger canister's parameters.

> For now, the only ledger parameters that can be changed is the transfer fee, token symbol, token name, and token logo.

Reviewer: ```suggestion
The ledger parameters that can be changed are the transfer fee, the token symbol, the token name, and the token logo.
```

Later, additional paramters might be added.
Fields where a value is set to `None` will remain unchanged.

#### Relevant type signatures

```candid
type ManageLedgerParameters = record {
  token_symbol : opt text;
  transfer_fee : opt nat64;
  token_logo : opt text;
  token_name : opt text;
};
```

> Example in bash (only change the transfer fee):

Reviewer 2: ```suggestion
Example in bash that only changes the transfer fee:
```
